src/character.cpp

```diff
@@ -231,7 +231,7 @@ void Character::recalc_sight_limits()
     } else if (has_active_mutation("SHELL2")) {
         // You can kinda see out a bit.
         sight_max = 2;
-    } else if ( (has_trait("MYOPIC") || has_trait("URSINE_EYE")) &&
+    } else if ( (has_trait("MYOPIC") || has_active_mutation("URSINE_EYE")) &&
```

Comment:
Yeah, you're gonna be nearsighted if you've got bear eyes.  Need glasses/contacts to turn that off.  ;-)